pkg/util/kubeconfig/kubeconfig.go

```diff
@@ -70,6 +73,10 @@ func WriteKubeConfig(ctx context.Context, currentNamespaceClient client.Client,
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      secretName,
 				Namespace: secretNamespace,
+				Labels: map[string]string{
+					KubeConfigSecretLabelAppKey:     KubeConfigSecretLabelAppValue,
+					KubeConfigSecretLabelReleaseKey: secretNamespace,
```

Comment:
I don't think the release name is equal to the namespace name, that only happens for development where the release is equal the namespace

Reply:
Hi @FabianKramm, I have made the necessary changes. I initially wanted to use 'vcluster.loft.sh/vcluster-name' instead of 'vcluster-name' as the key, but found the same key used as an annotation in syncer.go